question_framework/validation/validation.py

```diff
@@ -57,3 +57,17 @@ def ip_range(x: str) -> bool:
         return False
     ip1, ip2 = split_ips
     return is_ip_check(ip1) and is_ip_check(ip2)
+
+def date_validation(x):
+  date_format = '%Y-%m-%d'
+  try:
+    date_obj = datetime.strptime(x, date_format)
```

Comment:
try checking out this stack overflow for ways to validate
https://stackoverflow.com/questions/16870663/how-do-i-validate-a-date-string-format-in-python/16870682

Reply:
I will correct it.